static/js/click_handlers.js

```diff
@@ -659,9 +662,9 @@ exports.initialize = function () {
         popovers.hide_mobile_message_buttons_popover();
         compose_actions.start("private");
     });
-
     $(".compose_reply_button").on("click", () => {
         compose_actions.respond_to_message({trigger: "reply button"});
+        gear_menu.close_if_needed();
```

Comment:
How did you pick the set of buttons to add this to?

Reply:
The same logic goes when the user presses on any of the <code>ComposeBox</code> buttons as : 
<strong>Image</strong>
![Screenshot from 2020-09-24 12-40-50](https://user-images.githubusercontent.com/53977614/94112521-36090680-fe63-11ea-86ae-d9beaa5411a5.png)